modules/visual_script/visual_script_editor.cpp

```diff
@@ -3108,9 +3140,11 @@ void VisualScriptEditor::_member_rmb_selected(const Vector2 &p_pos) {
 
 		member_type = MEMBER_SIGNAL;
 		member_name = ti->get_text(0);
-		member_popup->add_icon_item(edit_icon, TTR("Edit Signal"), MEMBER_EDIT);
+		member_popup->add_icon_shortcut(edit_icon, ED_GET_SHORTCUT("visual_script_editor/edit_member"), MEMBER_EDIT);
+		member_popup->set_item_text(member_popup->get_item_count() - 1, "Edit Signal");
 		member_popup->add_separator();
-		member_popup->add_icon_item(del_icon, TTR("Remove Signal"), MEMBER_REMOVE);
+		member_popup->add_icon_shortcut(del_icon, ED_GET_SHORTCUT("visual_script_editor/delete_selected"), MEMBER_REMOVE);
+		member_popup->set_item_text(member_popup->get_item_count() - 1, "Remove Signal");
```

Comment:
Why you removed the localization `TTR()`?
Lines [`3121`](https://github.com/godotengine/godot/pull/15379/files/2f97af36cbde5379c6b739b4f2dc419fa38fc5bb#diff-bc4de62b8d91eb9cc820c6cce706d8b3R3121), [`3131`](https://github.com/godotengine/godot/pull/15379/files/2f97af36cbde5379c6b739b4f2dc419fa38fc5bb#diff-bc4de62b8d91eb9cc820c6cce706d8b3R3131), [`3134`](https://github.com/godotengine/godot/pull/15379/files/2f97af36cbde5379c6b739b4f2dc419fa38fc5bb#diff-bc4de62b8d91eb9cc820c6cce706d8b3R3134), [`3144`](https://github.com/godotengine/godot/pull/15379/files/2f97af36cbde5379c6b739b4f2dc419fa38fc5bb#diff-bc4de62b8d91eb9cc820c6cce706d8b3R3144), [`3147`](https://github.com/godotengine/godot/pull/15379/files/2f97af36cbde5379c6b739b4f2dc419fa38fc5bb#diff-bc4de62b8d91eb9cc820c6cce706d8b3R3147).

Reply:
My bad, those were supposed to be deleted since they don't actually work.